### ISSUE TITLE
Inform basic members about limitation for attribute filter

### ISSUE THREAD

**Issue by Lineflyer** [cgeo/cgeo#13007]
### Describe your problem!

I had a beta user on support, who hoped to be able to use the attribute filtering now on live map.
But despite he activates filters for certain attributes all caches keep showing.

I remember something about limitations for basic members here, am I right?

If this is the case, we should maybe add some kind of hint to that filter, that it only works on stored caches for basic members.

### How to reproduce?

- Basic member of gc.com
- Open live map
- Set advanced filter for a certain attribute (here: challenge caches)

### Actual result after these steps?

Still all caches shown

### Expected result after these steps?

Only challenge caches shown

### Reproducible

Yes

### c:geo Version

2022.05.09-RC

### System information

```text
Note: Filter was not set when taking the sys info, but I checked on a users screenshot

## System information

c:geo version: 2022.05.09-RC

Device:
-------
- Device type: SM-G980F (x1seea, samsung)
- Available processors: 8
- Android version: 12
- Android build: SP1A.210812.016.G980FXXSDFVC9
- Screen resolution: 1080x2196px (411x836dp)
- Pixel density: 2.8875
- System font scale: 1.1 / used scale: 1.1
- Sailfish OS detected: false
- Google Play services: enabled - 22.15.14 (190408-441847897)

Sensor and location:
-------
- Low power mode: inactive
- Compass capabilities: yes
- Rotation vector sensor: present
- Orientation sensor: present
- Magnetometer & Accelerometer sensor: present
- Direction sensor used: rotation vector

Program settings:
-------
- Settings: v9, Count:130
- Set language: en
- System date format: y. MM. dd.
- Time zone: GMT+02:00
- Debug mode active: no
- Last backup: never
- Routing mode: Walk
- Live map mode: true
- OSM multi-threading: false / threads: 1
- Map: OpenStreetMap.org
  - Id: cgeo.geocaching.maps.mapsforge.MapsforgeMapProvider$OsmMapSource
  - Atts: OpenStreetMap DE, map data OpenStreetMap contributors
  - Theme: none

Filters:
-------
- Hide waypoints: original parking visited
- LIVE: Állapot:Found=No ([:inconclusive=false:advanced=true]AND(status:found_no))
- OFFLINE: Semelyik ([:inconclusive=false:advanced=false])

Stored Filters (#0):

Services:
-------
- Geocaching sites enabled:
   geocaching.com: Logged in (Bejelentkezés ok) / BASIC
- Geocaching.com date format: yyyy.MM.dd.
- Routing: external / BRouter installed: false
- Installed c:geo plugins:  none

Permissions & paths:
-------
- Fine location permission: granted
- Write external storage permission: granted
- System internal c:geo dir: /data/user/0/cgeo.geocaching (34,2 GB free) v3 internal isDir(99 entries)
- Legacy User storage c:geo dir: /storage/emulated/0/cgeo (34,2 GB free) v3 external non-removable isDir(1 entries)
- Geocache data: /storage/emulated/0/Android/data/cgeo.geocaching/files/GeocacheData (34,2 GB free) v3 external non-removable isDir(76 entries)
- Internal theme sync (is turned off): /data/user/0/cgeo.geocaching/MapThemeData (34,2 GB free) v3 internal isDir(0 entries)
- Public Folders: #11
  - BASE: /cgeo/gpx (User-Defined)[/cgeo/gpx[DOCUMENT#0:p-content://com.android.externalstorage.documents/tree/primary%3Acgeo%2Fgpx::]] (Uri: content://com.android.externalstorage.documents/tree/primary%3Acgeo%2Fgpx/document/primary%3Acgeo%2Fgpx, Av:true, files:>=1, dirs:>=8, totalFileSize:>=579,9 KB, free space: 34,2 GB, files on device: 13511675)
  - OFFLINE_MAPS: /cgeo/gpx/maps (Default)[/cgeo/gpx/maps[PERSISTABLE_FOLDER(BASE)#1:p-content://com.android.externalstorage.documents/tree/primary%3Acgeo%2Fgpx::/maps]] (Uri: content://com.android.externalstorage.documents/tree/primary%3Acgeo%2Fgpx/document/primary%3Acgeo%2Fgpx%2Fmaps, Av:true, files:0, dirs:1, totalFileSize:0 B, free space: 34,2 GB, files on device: 13511675)
  - OFFLINE_MAP_THEMES: /cgeo/gpx/maps/_themes (Default)[/cgeo/gpx/maps/_themes[PERSISTABLE_FOLDER(OFFLINE_MAPS)#1:p-content://com.android.externalstorage.documents/tree/primary%3Acgeo%2Fgpx::/maps/_themes]] (Uri: content://com.android.externalstorage.documents/tree/primary%3Acgeo%2Fgpx/document/primary%3Acgeo%2Fgpx%2Fmaps%2F_themes, Av:true, files:0, dirs:0, totalFileSize:0 B, free space: 34,2 GB, files on device: 13511675)
  - LOGFILES: /cgeo/gpx/logfiles (Default)[/cgeo/gpx/logfiles[PERSISTABLE_FOLDER(BASE)#1:p-content://com.android.externalstorage.documents/tree/primary%3Acgeo%2Fgpx::/logfiles]] (Uri: content://com.android.externalstorage.documents/tree/primary%3Acgeo%2Fgpx/document/primary%3Acgeo%2Fgpx%2Flogfiles, Av:true, files:1, dirs:0, totalFileSize:579,9 KB, free space: 34,2 GB, files on device: 13511675)
  - GPX: /cgeo/gpx/gpx (Default)[/cgeo/gpx/gpx[PERSISTABLE_FOLDER(BASE)#1:p-content://com.android.externalstorage.documents/tree/primary%3Acgeo%2Fgpx::/gpx]] (Uri: content://com.android.externalstorage.documents/tree/primary%3Acgeo%2Fgpx/document/primary%3Acgeo%2Fgpx%2Fgpx, Av:true, files:0, dirs:0, totalFileSize:0 B, free space: 34,2 GB, files on device: 13511675)
  - BACKUP: /cgeo/gpx/backup (Default)[/cgeo/gpx/backup[PERSISTABLE_FOLDER(BASE)#1:p-content://com.android.externalstorage.documents/tree/primary%3Acgeo%2Fgpx::/backup]] (Uri: content://com.android.externalstorage.documents/tree/primary%3Acgeo%2Fgpx/document/primary%3Acgeo%2Fgpx%2Fbackup, Av:true, files:0, dirs:0, totalFileSize:0 B, free space: 34,2 GB, files on device: 13511675)
  - FIELD_NOTES: /cgeo/gpx/field-notes (Default)[/cgeo/gpx/field-notes[PERSISTABLE_FOLDER(BASE)#1:p-content://com.android.externalstorage.documents/tree/primary%3Acgeo%2Fgpx::/field-notes]] (Uri: content://com.android.externalstorage.documents/tree/primary%3Acgeo%2Fgpx/document/primary%3Acgeo%2Fgpx%2Ffield-notes, Av:true, files:0, dirs:0, totalFileSize:0 B, free space: 34,2 GB, files on device: 13511675)
  - SPOILER_IMAGES: /cgeo/gpx/GeocachePhotos (Default)[/cgeo/gpx/GeocachePhotos[PERSISTABLE_FOLDER(BASE)#1:p-content://com.android.externalstorage.documents/tree/primary%3Acgeo%2Fgpx::/GeocachePhotos]] (Uri: content://com.android.externalstorage.documents/tree/primary%3Acgeo%2Fgpx/document/primary%3Acgeo%2Fgpx%2FGeocachePhotos, Av:true, files:0, dirs:0, totalFileSize:0 B, free space: 34,2 GB, files on device: 13511675)
  - ROUTING_BASE: /cgeo/gpx/routing (Default)[/cgeo/gpx/routing[PERSISTABLE_FOLDER(BASE)#1:p-content://com.android.externalstorage.documents/tree/primary%3Acgeo%2Fgpx::/routing]] (Uri: content://com.android.externalstorage.documents/tree/primary%3Acgeo%2Fgpx/document/primary%3Acgeo%2Fgpx%2Frouting, Av:true, files:7, dirs:1, totalFileSize:81,2 KB, free space: 34,2 GB, files on device: 13511675)
  - ROUTING_TILES: /cgeo/gpx/routing/segments4 (Default)[/cgeo/gpx/routing/segments4[PERSISTABLE_FOLDER(ROUTING_BASE)#1:p-content://com.android.externalstorage.documents/tree/primary%3Acgeo%2Fgpx::/routing/segments4]] (Uri: content://com.android.externalstorage.documents/tree/primary%3Acgeo%2Fgpx/document/primary%3Acgeo%2Fgpx%2Frouting%2Fsegments4, Av:true, files:0, dirs:0, totalFileSize:0 B, free space: 34,2 GB, files on device: 13511675)
  - TEST_FOLDER: [Legacy]/data/user/0/cgeo.geocaching/files/unittest (Default)[/data/user/0/cgeo.geocaching/files/unittest[FILE#1:p-file:///data/user/0/cgeo.geocaching/files::/unittest]] (Uri: file:///data/user/0/cgeo.geocaching/files/unittest, Av:true, files:0, dirs:0, totalFileSize:0 B, free space: 34,2 GB, files on device: -1)
- Map render theme path:
- PersistedDocumentUris: #1
- UNUSED_TRACK: null
- Persisted Uri Permissions: #1
  - content://com.android.externalstorage.documents/tree/primary%3Acgeo%2Fgpx (2021. ápr. 19. 7:41):RW
- Database: /data/user/0/cgeo.geocaching/databases/data (v98, Size:1,9 MB) on system internal storage

--- End of system information ---
```

### Additional Information

Support ticket #846656

**Comment by eddiemuc** [comment]
Indeed, works only for premium members

**Comment by Lineflyer** [comment]
As mentioned above we could maybe add a hint text into the attribute filter dialog, that functionality is limited if membertype=Basic. Changed issue title accordingly.

**Comment by murggel** [comment]
Closed as duplicate to #14577
